core/trino-main/src/main/java/io/trino/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -806,6 +807,39 @@ private static String getFunctionType(FunctionMetadata function)
             };
         }
 
+        @Override
+        protected Node visitShowBranches(ShowBranches showColumns, Void context)
+        {
+            QualifiedObjectName tableName = createQualifiedObjectName(session, showColumns, showColumns.getTableName());
+            getRequiredCatalogHandle(metadata, session, showColumns, tableName.catalogName());
+            if (!metadata.schemaExists(session, new CatalogSchemaName(tableName.catalogName(), tableName.schemaName()))) {
```

Comment:
Do we need to check if schema exists if we check table later on?

Reply:
This check exists for better error message.